config/jobs/containerd/containerd/containerd-presubmit-jobs.yaml

```diff
@@ -0,0 +1,92 @@
+    annotations:
+      testgrid-dashboards: sig-node-containerd-io
+      testgrid-tab-name: pull-containerd-build
+      description: build artifacts
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200901-eeeadc5-master
+        command:
+        - runner.sh
+        args:
+        - ./test/build.sh
+
+  - name: pull-containerd-node-e2e
+    always_run: false
+    optional: true
+    max_concurrency: 8
+    decorate: true
+    branches:
+    - master
+    decoration_config:
+      timeout: 100m
+    extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
+    annotations:
+      testgrid-dashboards: sig-node-containerd-io
+      testgrid-tab-name: pull-containerd-node-e2e
+      description: run node e2e tests
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - name: pull-containerd-node-e2e
+        image: gcr.io/k8s-testimages/kubekins-e2e:v20200901-eeeadc5-master
+        command:
+        - sh
+        - -c
+        - >
+          runner.sh
+          ./test/build.sh
+          &&
+          cd ${GOPATH}/src/k8s.io/kubernetes
+          &&
+          /workspace/scenarios/kubernetes_e2e.py
+          --deployment=node
+          --gcp-project=cri-c8d-pr-node-e2e
+          --gcp-zone=us-central1-f
+          '--node-test-args=--container-runtime=remote --container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/home/containerd/usr/local/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd\"]}"'
+          --node-tests=true
+          --provider=gce
+          '--test_args=--nodes=8 --focus="\[NodeConformance\]|\[NodeFeature:.+\]" --skip="\[Flaky\]|\[Slow\]|\[Serial\]"'
+          --timeout=65m
+          "--node-args=--image-config-file=${GOPATH}/src/k8s.io/test-infra/jobs/e2e_node/containerd/cri-master/image-config-pr.yaml -node-env=PULL_REFS=$(PULL_REFS)"
+
+  - name: pull-containerd-verify
+    always_run: false
+    optional: true
+    branches:
+    - master
+    annotations:
+      testgrid-dashboards: sig-node-containerd-io
+      testgrid-tab-name: pull-containerd-verify
+      description: verify job
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200901-eeeadc5-master
+        args:
+        - "--repo=github.com/containerd/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - --scenario=execute
+        - --
+        - ./test/verify.sh
```

Comment:
..

Reply:
same as above. we can figure out if we need this and add a script or switch make targets